lib/active_model_serializers/adapter/base.rb

```diff
@@ -41,6 +43,10 @@ def cache_check(serializer)
 
       private
 
+      def serialization_options(options)
+        (options ||= {}).slice(*ActiveModel::Serializer::SERIALIZABLE_HASH_VALID_KEYS) # rubocop:disable Lint/UselessAssignment
```

Comment:
I guess in the current implementation it doesn't matter. Good eyes. previously, when it was inline, we, of course wanted to set the value of options when nil.